Checkbox/Checkbox.module.less

```diff
@@ -0,0 +1,35 @@
+// Checkbox.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.toggleIcon {
+	.applySkins({
+		@size: @agate-checkbox-size;
+
+		width: @size;
+		height: @size;
+		border-radius: @agate-checkbox-border-radius;
+
+		.icon {
+			visibility: hidden;
+			margin: 0;
+			line-height: @size;
+			width: @size;
+			height: @size;
+		}
+
+		&.selected {
+			.icon {
+				visibility: visible;
+			}
+		}
+
+		color: @agate-checkbox-text-color;
```

Comment:
We've got some disjointed rules here. Let's move these up with the others and collapse the two `.icon` blocks.